src/Tribe/Views/V2/Hooks.php

```diff
@@ -581,4 +582,50 @@ public function filter_events_properties( $event ) {
 
 		return $this->container->make( Template\Event::class )->filter_event_properties( $event );
 	}
+
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$context" missing

<sub>hash: c8a0210102e80715d4dcae56690421c9</sub>